### ISSUE TITLE
about the training data of the the pretraind model

### ISSUE THREAD

**Issue by yustaub** [open-mmlab/OpenPCDet#331]
hello, what is the training data of your provided pretrained kitti model, do you split the kitti 3d detection training dataset into train and val? I don't understand `The results are the 3D detection performance of moderate difficulty on the val set of KITTI dataset` the **val** set, very appreciate for your reply.

**Comment by jihanyang** [comment]
Yes. We divide the kitti offical training set into training set and validation set as our `data/kitti/ImageSets`. The results in our readme are evaluated in validation set.

**Comment by yustaub** [comment]
ok, thanks for your reply!! @jihanyang
